Brokerages/Bitfinex/BitfinexBrokerage.cs

```diff
@@ -269,7 +273,9 @@ public override List<CashAmount> GetCashBalance()
 
             if (response.StatusCode != HttpStatusCode.OK)
             {
-                throw new Exception($"BitfinexBrokerage.GetCashBalance: request failed: [{(int)response.StatusCode}] {response.StatusDescription}, Content: {response.Content}, ErrorMessage: {response.ErrorMessage}");
+                throw new Exception($"BitfinexBrokerage.GetCashBalance: request failed: " +
+                                    $"[{(int)response.StatusCode}] {response.StatusDescription}, " +
+                                    $"Content: {response.Content}, ErrorMessage: {response.ErrorMessage}");
```

Comment:
Just a general comment: should avoid format/style changes like these in PRs with other objectives, like fixing a bug.